src/libraries/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.File.Changed.cs

```diff
@@ -27,32 +26,30 @@ public void FileSystemWatcher_File_Changed_LastWrite()
         [InlineData(true)]
         public void FileSystemWatcher_File_Changed_Nested(bool includeSubdirectories)
         {
-            using (var dir = new TempDirectory(GetTestFilePath()))
-            using (var firstDir = new TempDirectory(Path.Combine(dir.Path, "dir1")))
-            using (var nestedDir = new TempDirectory(Path.Combine(firstDir.Path, "nested")))
-            using (var nestedFile = new TempFile(Path.Combine(nestedDir.Path, "nestedFile")))
-            using (var watcher = new FileSystemWatcher(dir.Path, "*"))
+            string firstDir = Path.Combine(TestDirectory, "dir1");
+            string nestedDir = CreateTestDirectory(firstDir, "nested");
+            string nestedFile = CreateTestFile(nestedDir, "nestedFile");
```

Comment:
```suggestion
            string nestedFile = CreateTestFile(TestDirectory, "dir1", "nested", "nestedFile");
```
firstDir, nestedDir are not used, and we made CreateTestFile create any directories it needs.